src/main/kotlin/io/snyk/plugin/ui/actions/SnykCleanScanAction.kt

```diff
@@ -19,4 +20,8 @@ class SnykCleanScanAction : AnAction(AllIcons.Actions.GC), DumbAware {
         val project = actionEvent.project
         actionEvent.presentation.isEnabled = project != null && !project.isDisposed
     }
+
+    override fun getActionUpdateThread(): ActionUpdateThread {
+        return ActionUpdateThread.BGT
```

Comment:
BGT short for background thread I think.

Reply:
Yep. The JavaDoc states this:
![image](https://github.com/snyk/snyk-intellij-plugin/assets/20150761/856e0c92-a633-42cf-ada6-2cd37baf219c)